protoc-gen-jersey/src/main/java/com/fullcontact/rpc/jersey/CodeGenerator.java

```diff
@@ -85,6 +83,16 @@
             for(Descriptors.ServiceDescriptor serviceDescriptor : fd.getServices()) {
                 DescriptorProtos.ServiceDescriptorProto serviceDescriptorProto = serviceDescriptor.toProto();
                 for(DescriptorProtos.MethodDescriptorProto methodProto : serviceDescriptorProto.getMethodList()) {
+                    String fullMethodName = serviceDescriptor.getFullName() +"." + methodProto.getName();
+                    if(yamlConfig.isPresent()){   //Check to see if the rules are defined in the YAML
+                        for(YamlHttpRule rule :yamlConfig.get().getRules()){
```

Comment:
Space after `:` -- I also noticed that most lines are lacking a space between `)` and `{`. I don't have a problem with it in spirit, but I would like to keep the style consistent.